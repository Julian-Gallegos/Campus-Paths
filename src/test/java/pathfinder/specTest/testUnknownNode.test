# Find a path with Dijkstra's algorithm when
# the starting node does not exist

CreateGraph simpleGraph

AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph C D 1.0
AddEdge simpleGraph D B 1.0

FindPath simpleGraph A B