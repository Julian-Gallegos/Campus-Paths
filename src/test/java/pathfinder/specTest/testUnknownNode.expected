# Find a path with Dijkstra's algorithm when
# the starting node does not exist

created graph simpleGraph

added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 1.000 from C to D in simpleGraph
added edge 1.000 from D to B in simpleGraph

path from A to B:
unknown node A